api/types/mount/mount.go

```diff
@@ -56,3 +60,12 @@ type Driver struct {
 	Name    string            `json:",omitempty"`
 	Options map[string]string `json:",omitempty"`
 }
+
+// TmpfsOptions defines options specific to mounts of type "tmpfs".
+type TmpfsOptions struct {
+	// RawOptions is the raw string passed to mount(2).
+	// e.g. "rw,noexec,nosuid,size=65536k"
+	// RawOptions can contain "ro" or "rw" but needs to be consistent with
+	// Mount.ReadOnly
+	RawOptions string `json:",omitempty"`
```

Comment:
ok, I'm going to change the struct definition to like this:
```go
// TmpfsOptions defines options specific to mounts of type "tmpfs".                                                                      
type TmpfsOptions struct {
        // Size sets the size of the tmpfs, in bytes.                                                                                    
        //                                                                                                                               
        // This will be converted to an operating system specific value                                                                  
        // depending on the host. For example, on linux, it will be convered to                                                          
        // use a 'k', 'm' or 'g' syntax. BSD, though not widely supported with                                                           
        // docker, uses a straight byte value.                                                                                           
        //                                                                                                                               
        // Percentages are not supported.                                                                                                
        SizeBytes int64
        // Mode of the tmpfs upon creation                                                                                               
        Mode os.FileMode

        // RawOptions is the raw string passed to mount(2).                                                                              
        // e.g. "rw,noexec,nosuid,size=65536k"                                                                                           
        //                                                                                                                               
        // RawOptions is exclusive to other TmpfsOptions fields.                                                                         
        // If RawOptions has a non-empty value, other fields must have empty values.                                                     
        //                                                                                                                               
        // RawOptions can contain "ro" or "rw" but needs to be consistent with                                                           
        // Mount.ReadOnly                                                                                                                
        RawOptions string `json:",omitempty"`
}
```

I think we need to keep RawOptions for compatibility, WDYT?